src/example-relay/.babelrc.js

```diff
@@ -1,13 +1,11 @@
 // @flow strict
 
 module.exports = {
-  presets: ['@adeira/babel-preset-adeira', 'next/babel'],
-  /**
-   * test-bc script broke after upgrading some babel packages.
-   * Adding @babel/plugin-proposal-class-properties made it work, but it should not be necessary
-   * to add it here, since we already have it in @adeira/babel-preset-adeira.
-   * We need to leave it here for now for the test-bc script to work, but we should try
-   * to remove it in the future
-   */
-  plugins: ['relay', '@babel/plugin-proposal-class-properties'],
+  presets: ['@adeira/babel-preset-adeira'],
+  // For some unknown reason, we have to specify here the `class-properties` Babel plugin explicitly
```

Comment:
Can we remove this comment now?

Reply:
I don't think so. Previously there was a comment about the `test-bc` script that doesn't exist anymore. But the issue is still valid and it affects the application itself as well. And it's still unclear to me why these two presets interfere, see: https://github.com/adeira/universe/issues/1854.